app/services/albums.js

```diff
@@ -25,7 +23,33 @@ const listAlbumPhotos = albumId => {
   });
 };
 
+const findById = albumId =>
+  models.albums.findByPk(albumId).catch(() => {
+    throw errors.databaseError('Album not found');
+  });
+
+const createAlbum = albumData =>
+  models.albums.create(albumData).catch(() => {
+    throw errors.databaseError('Could not create album');
+  });
+
+const getInfoById = albumId => {
+  options.uri = `${base_uri}/albums/${albumId}`;
+  return rp(options)
+    .then(result => {
+      const albumData = JSON.parse(result);
+      delete albumData.userId;
```

Comment:
Why did you delete this?

Reply:
When requesting for an album in the API, it comes back with a `userId` that's completely unrelated to our DBs user ids. Therefore I found it appropriate to delete it since that information is of no use.